src/getCoppyFile.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+
+const path = require('path');
+
+function getCopyFile(source, destination) {
+  const sourcePath = path.resolve(source);
+  const destPath = path.resolve(destination);
+
+  if (sourcePath === destPath) {
+    console.log('Source and destination are the same.');
+
+    return;
+  }
+
+  if (!fs.existsSync(sourcePath)) {
+    console.log(`Source file '${sourcePath}' does not exist.`);
+
+    return;
+  }
+
+  try {
+    fs.copyFileSync(sourcePath, destPath);
+    console.log(`'${sourcePath}' copied to '${destPath}' successfully.`);
+  } catch (err) {
+    console.log(`Error copying '${sourcePath}' to '${destPath}':
+    ${err.message}`);
```

Comment:
You can handle all these errors with a try-catch. Because function fs.copyFileSync provides all of these errors

> If the source file or directory does not exist, it will throw a ENOENT error.
> If the destination directory does not exist, it will throw a ENOENT error.
> If the destination is a directory that already exists, it will throw an EEXIST error.
> If the user does not have permission to read the source file or directory, it will throw a EACCES error.
> If the user does not have permission to write to the destination directory, it will throw a EACCES error.
> If the destination directory is on a different file system than the source file, it will throw a EXDEV error.